cfme/tests/configure/test_default_views_cloud.py

```diff
@@ -52,22 +56,7 @@ def set_and_test_default_view(appliance, group_name, expected_view, page):
     default_views.set_default_view(group_name, old_default, fieldset='Clouds')
 
 
-@pytest.mark.parametrize('key', gtl_params, scope="module")
-def test_cloud_tile_defaultview(appliance, request, key):
-    set_and_test_default_view(appliance, key, 'Tile View', gtl_params[key])
-
-
-@pytest.mark.parametrize('key', gtl_params, scope="module")
-def test_cloud_list_defaultview(appliance, request, key):
-    set_and_test_default_view(appliance, key, 'List View', gtl_params[key])
-
-
-@pytest.mark.parametrize('key', gtl_params, scope="module")
-def test_cloud_grid_defaultview(appliance, request, key):
-    set_and_test_default_view(appliance, key, 'Grid View', gtl_params[key])
-
-
-def set_and_test_compare_view(appliance, group_name, expected_view, selector_type='views_selector'):
+def set_and_test_compare_view(appliance, group_name, expected_view, selector_type):
```

Comment:
Please add doc string for this test too

Reply:
Please move assert statement from this method in to test 
assert expected_view == selected_view, '{} setting failed'.format(expected_view)